.golangci.yml

```diff
@@ -0,0 +1,192 @@
+    - revive # : Fast, configurable, extensible, flexible, and beautiful linter for Go. Drop-in replacement of golint. [fast: false, auto-fix: false]
+    - rowserrcheck # : checks whether Err of rows is checked successfully [fast: false, auto-fix: false]
+    - sqlclosecheck # : Checks that sql.Rows and sql.Stmt are closed. [fast: false, auto-fix: false]
+    - stylecheck # : Stylecheck is a replacement for golint [fast: false, auto-fix: false]
+    - tagliatelle # : Checks the struct tags. [fast: true, auto-fix: false]
+    - tenv #: tenv is analyzer that detects using os.Setenv instead of t.Setenv since Go1.17 [fast: false, auto-fix: false]
+    - testpackage # : linter that makes you use a separate _test package [fast: true, auto-fix: false]
+    - thelper # : thelper detects golang test helpers without t.Helper() call and checks the consistency of test helpers [fast: false, auto-fix: false]
+    - tparallel # : tparallel detects inappropriate usage of t.Parallel() method in your Go test codes [fast: false, auto-fix: false]
+    - unconvert # : Remove unnecessary type conversions [fast: false, auto-fix: false]
+    - unparam # : Reports unused function parameters [fast: false, auto-fix: false]
+    - varnamelen #: checks that the length of a variable's name matches its scope [fast: false, auto-fix: false]
+    - wastedassign # : wastedassign finds wasted assignment statements. [fast: false, auto-fix: false]
+    - whitespace # : Tool for detection of leading and trailing whitespace [fast: true, auto-fix: true]
+    - wrapcheck # : Checks that errors returned from external packages are wrapped [fast: false, auto-fix: false]
+    - wsl # : Whitespace Linter - Forces you to use empty lines! [fast: true, auto-fix: false]
+
+# output configuration options
+output:
+  # colored-line-number|line-number|json|tab|checkstyle|code-climate|junit-xml|github-actions
+  # default is "colored-line-number"
+  format: line-number
+
+  # print lines of code with issue, default is true
+  print-issued-lines: false
+
+  # print linter name in the end of issue text, default is true
+  print-linter-name: true
+
+  # make issues output unique by line, default is true
+  # uniq-by-line: true
+
+  # add a prefix to the output file references; default is no prefix
+  # path-prefix: ""
+
+  # sorts results by: filepath, line and column
+  sort-results: false
+
+# all available settings of specific linters
+linters-settings:
+  gocritic:
+    # Enable multiple checks by tags, run `GL_DEBUG=gocritic golangci-lint run` to see all tags and checks.
+    # Empty list by default. See https://github.com/go-critic/go-critic#usage -> section "Tags".
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - hugeParam # does not work with generics
+      - typeDefFirst # does not work with generics
+
+  gocyclo:
```

Comment:
Is it possible not to run gocyclo on tests?
https://github.com/pantafive/qualifying-grade/runs/7565832262?check_suite_focus=true#step:4:25

Reply:
Done